coverage/block-navigation.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
```

Comment:
### Critical

- Include the coverage directory inside `.gitignore`

Reply:
yeah I have done